submanifests/optional.yaml

```diff
@@ -27,6 +29,12 @@ manifest:
       remote: upstream
       groups:
         - optional
+    - name: nrf_wifi
```

Comment:
@krish2718 as per a discussion yesterday in the TSC, please move this to the root's `west.yml`.